drivers/input/touchscreen/rpi-ft5406.c

```diff
@@ -81,18 +84,21 @@ static int ft5406_thread(void *arg)
 				int x = (((int) regs.point[i].xh & 0xf) << 8) + regs.point[i].xl;
 				int y = (((int) regs.point[i].yh & 0xf) << 8) + regs.point[i].yl;
 				int touchid = (regs.point[i].yh >> 4) & 0xf;
+				int event_id = (regs.point[i].xh >> 6) & 0x3;
 				
 				modified_ids |= 1 << touchid;
 
-				if(!((1 << touchid) & known_ids))
-					dev_dbg(&ts->pdev->dev, "x = %d, y = %d, touchid = %d\n", x, y, touchid);
-				
-				input_mt_slot(ts->input_dev, touchid);
-				input_mt_report_slot_state(ts->input_dev, MT_TOOL_FINGER, 1);
+				if (event_id == FTS_TOUCH_CONTACT || event_id == FTS_TOUCH_DOWN)
+				{
```

Comment:
This brace belongs on the previous line. If it doesn't fit, split the predicate after the `||`.